[BUG] Right Clicking Parallel Hatch throws an error

**Describe the bug**
It only appears on server
```
[22:09:56] [Server thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NoClassDefFoundError: net/minecraft/client/resources/I18n
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_312]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_312]
	at net.minecraft.util.Util.runTask(SourceFile:47) [h.class:?]
	at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:723) [MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.updateTimeLightAndEntities(DedicatedServer.java:397) [nz.class:?]
	at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/resources/I18n
	at gregicality.multiblocks.common.metatileentities.multiblockpart.MetaTileEntityParallelHatch.createUI(MetaTileEntityParallelHatch.java:68) ~[MetaTileEntityParallelHatch.class:?]
	at gregtech.api.metatileentity.MetaTileEntityUIFactory.createUITemplate(MetaTileEntityUIFactory.java:31) ~[MetaTileEntityUIFactory.class:?]
	at gregtech.api.metatileentity.MetaTileEntityUIFactory.createUITemplate(MetaTileEntityUIFactory.java:18) ~[MetaTileEntityUIFactory.class:?]
	at gregtech.api.gui.UIFactory.openUI(UIFactory.java:36) ~[UIFactory.class:?]
	at gregtech.api.metatileentity.MetaTileEntity.onRightClick(MetaTileEntity.java:386) ~[MetaTileEntity.class:?]
	at gregtech.api.metatileentity.MetaTileEntity.onCoverRightClick(MetaTileEntity.java:359) ~[MetaTileEntity.class:?]
	at gregtech.api.block.machines.BlockMachine.onBlockActivated(BlockMachine.java:334) ~[BlockMachine.class:?]
	at net.minecraft.server.management.PlayerInteractionManager.processRightClickBlock(PlayerInteractionManager.java:449) ~[or.class:?]
	at net.minecraft.network.NetHandlerPlayServer.processTryUseItemOnBlock(NetHandlerPlayServer.java:741) ~[pa.class:?]
	at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.processPacket(SourceFile:55) ~[ma.class:?]
	at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.processPacket(SourceFile:11) ~[ma.class:?]
	at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) ~[hv$1.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_312]
	at net.minecraft.util.Util.runTask(SourceFile:46) ~[h.class:?]
	... 5 more
```

**Versions**
GTCEu: 2.3.3
GCYM: 1.1.3

**Setup**
Server

--- a/src/main/java/gregicality/multiblocks/common/metatileentities/multiblockpart/MetaTileEntityParallelHatch.java
+++ b/src/main/java/gregicality/multiblocks/common/metatileentities/multiblockpart/MetaTileEntityParallelHatch.java
@@ -65,7 +65,7 @@ public class MetaTileEntityParallelHatch extends MetaTileEntityMultiblockPart im
     protected ModularUI createUI(@Nonnull EntityPlayer entityPlayer) {
         ServerWidgetGroup parallelAmountGroup = new ServerWidgetGroup(() -> true);
         parallelAmountGroup.addWidget(new ImageWidget(62, 36, 53, 20, GuiTextures.DISPLAY)
-                .setTooltip(I18n.format("gcym.machine.parallel_hatch.display")));
+                .setTooltip("gcym.machine.parallel_hatch.display"));
 
         parallelAmountGroup.addWidget(new IncrementButtonWidget(118, 36, 30, 20, 1, 4, 16, 64, this::setCurrentParallel)
                 .setDefaultTooltip()

